gui/src/app/util/triggerDownload.ts

```diff
@@ -0,0 +1,10 @@
+
+export const triggerDownload = (blob: Blob, name: string, onClose: () => void) => {
```

Comment:
This can also be used in SamplerOutputViews, I believe

Reply:
I knew it'd come in handy somewhere :)

Reply:
Pushed an update to use it there as well.